modules/search/instant-search/components/scroll-button.jsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+import { __ } from '@wordpress/i18n';
+// NOTE: We only import the debounce package here for to reduced bundle size.
+//       Do not import the entire lodash library!
+// eslint-disable-next-line lodash/import-scope
+import debounce from 'lodash/debounce';
+
+class ScrollButton extends Component {
+	componentDidMount() {
+		this.props.enableAutoScrolling && document.addEventListener( 'scroll', this.checkScroll );
+	}
+	componentDidUnmount() {
+		document.removeEventListener( 'scroll', this.checkScroll );
+	}
+
+	checkScroll = debounce( () => {
+		if (
+			this.props.enableAutoScrolling &&
+			window.innerHeight + window.scrollY === document.body.offsetHeight
+		) {
+			this.props.onClick();
+		}
+	}, 100 );
+
+	render() {
+		return (
+			<button
+				className="jetpack-instant-search__more-button"
+				disabled={ this.props.isLoading }
+				onclick={ this.props.onClick }
+			>
+				{ this.props.isLoading ? (
+					<span>{ __( 'Loading results…', 'jetpack' ) }</span>
+				) : (
+					<span>{ __( 'Load more results', 'jetpack' ) }</span>
```

Comment:
Feel like we should shorten the text here on the scroll button, but I'm not sure to what. "Load More"? "More"? Or could provide more details about how many more results there are?

cc @joanrho

Reply:
"Load more" is pretty standard.